src/main/java/org/fcrepo/importexport/exporter/Exporter.java

```diff
@@ -129,6 +135,8 @@ public Exporter(final Config config, final FcrepoClient.FcrepoClientBuilder clie
                 bagdir.mkdirs();
                 this.bag = new Bag(new Version(0, 97));
                 this.bag.setRootDir(bagdir);
+                this.bag.getMetadata().putAll(bagConfig.getBagInfo());
+                //TODO where to add aptrust bag info from the bagConfig?
```

Comment:
@escowles :  so are you suggesting that we just write in our own aptrust-info.txt filling it with kv pairs from the bagConfig.getAPTrustInfo()?